packages/web-runtime/src/components/Topbar/TopBar.vue

```diff
@@ -301,3 +301,4 @@ export default {
   }
 }
 </style>
+@ownclouders/web-pkg/src/composables/piniaStores/theme
```

Comment:
?  😋

Reply:
I knew that I had some weird miss-typing somewhere but couldn't find it anymore, thanks :D